SPECS/kata-containers-cc/kata-containers-cc.spec

```diff
@@ -43,6 +43,12 @@ Summary:        Kata Confidential Containers tools package for building the UVM
 %description tools
 This package contains the scripts and files required to build the UVM
 
+%package tardev
+Summary:        tardev-snapshotter for Kata Containers
```

Comment:
tardev-snapshotter is not necessarily tied to kata containers. we should rephrase to express its more generic purpose